Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift

```diff
@@ -20,7 +20,7 @@
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
```

Comment:
should we exclude vendor code?

Reply:
I did for most things. I can exclude here too but seemed harmless for now.